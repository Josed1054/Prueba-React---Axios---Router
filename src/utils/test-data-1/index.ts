export const Data = {
  _id: "5952983359954a0adbf7ab09",
  cityid: "MXAS0002",
  validdateutc: "20170627T140000Z",
  winddirectioncardinal: "SSE",
  probabilityofprecip: "40",
  relativehumidity: "90",
  name: "Aguascalientes",
  date_insert: "2017-06-27T17:36:43.084Z",
  longitude: "-102.296",
  state: "Aguascalientes",
  lastreporttime: "20170627T092449Z",
  skydescriptionlong: "Tormentas dispersas",
  stateabbr: "AGU",
  tempc: "17",
  latitude: "21.87982",
  iconcode: "96",
  windspeedkm: "6",
};
