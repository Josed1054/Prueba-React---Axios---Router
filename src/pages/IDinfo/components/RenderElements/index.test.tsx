import React from "react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import { RenderElements } from ".";
import { Data } from "../../../../utils/test-data-1/index";

test("Checar si se render la info", () => {
  const view = renderer.create(<RenderElements {...Data} />).toJSON();

  expect(view).toMatchSnapshot();
});
