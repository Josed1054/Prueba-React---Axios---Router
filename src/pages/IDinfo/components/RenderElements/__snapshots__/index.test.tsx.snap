// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checar si se render la info 1`] = `
<div>
  <p>
    _id: 5952983359954a0adbf7ab09
  </p>
  <p>
    cityid: MXAS0002
  </p>
  <p>
    validdateutc: 20170627T140000Z
  </p>
  <p>
    winddirectioncardinal: SSE
  </p>
  <p>
    probabilityofprecip: 40
  </p>
  <p>
    relativehumidity: 90
  </p>
  <p>
    name: Aguascalientes
  </p>
  <p>
    date_insert: 2017-06-27T17:36:43.084Z
  </p>
  <p>
    longitude: -102.296
  </p>
  <p>
    state: Aguascalientes
  </p>
  <p>
    lastreporttime: 20170627T092449Z
  </p>
  <p>
    skydescriptionlong: Tormentas dispersas
  </p>
  <p>
    stateabbr: AGU
  </p>
  <p>
    tempc: 17
  </p>
  <p>
    latitude: 21.87982
  </p>
  <p>
    iconcode: 96
  </p>
  <p>
    windspeedkm: 6
  </p>
</div>
`;
